src/test/java/tests/PickerMilestoneTest.java

```diff
@@ -17,8 +17,7 @@
     private static final String REPO_ID = "test/testrepo";
 
     private static List<PickerMilestone> listWithNoExistingAndSelected, listWithNoDefaultMilestone,
```

Comment:
Do we still need `listWithNoDefaultMilestone` since `default milestone` is being removed?